travis/apis.py

```diff
@@ -54,6 +56,19 @@ def __init__(self):
         self.ssh = os.environ.get(
             "WEBLATE_SSH", "ssh://user@webpage.com")
         self.tempdir = os.path.join(tempfile.gettempdir(), 'weblate_api')
+        self._ssh_keyscan()
+
+    def _ssh_keyscan(self):
+        """This method execute the command 'ssh-keysan' to avoid the
+        question when the command git clone is excecuted.
+        The question is like to:
+            'Are you sure you want to continue connecting (yes/no)?'"""
+        cmd = ['ssh-keyscan', '-p']
+        match = re.search(r'@([^/]+(?=:|/]))?(?::)?([^/]+)?/', self.ssh)
```

Comment:
Better use current method to split values of a url:
https://github.com/Vauxoo/maintainer-quality-tools/blob/548cce90f86e23a52e0ba8ef625de86508de19e3/travis/git_update.py#L52

Or something similar to get parameters with name from regex

Reply:
@moylop260 
I try to use the expression regular existing within the function "get_repo_data" but this not work as should be

![image](https://user-images.githubusercontent.com/1387970/30464124-a7961d2a-999d-11e7-8824-d38983461a5d.png)

I add group name to the regular expression existing in this code

![image](https://user-images.githubusercontent.com/1387970/30464001-fd1d232a-999c-11e7-859a-6016cbefecb4.png)

Only one trick the function "sorted" is required to get first the port and the second one the host

Reply:
@moylop260 
I change the regular expression to another one better